src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/roll.cpp

```diff
@@ -0,0 +1,102 @@
+                                           ::testing::Values(std::vector<int64_t>{0, 1}),      // Axes
+                                           ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+const auto testCase3D = ::testing::Combine(::testing::Values(std::vector<size_t>{2, 320, 320}),  // Input shape
+                                           ::testing::ValuesIn(inputPrecision),                  // Precision
+                                           ::testing::Values(std::vector<int64_t>{160, 160}),    // Shift
+                                           ::testing::Values(std::vector<int64_t>{1, 2}),        // Axes
+                                           ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+const auto testCaseNegativeUnorderedAxes4D =
+    ::testing::Combine(::testing::Values(std::vector<size_t>{3, 11, 6, 4}),  // Input shape
+                       ::testing::ValuesIn(inputPrecision),                  // Precision
+                       ::testing::Values(std::vector<int64_t>{7, 3}),        // Shift
+                       ::testing::Values(std::vector<int64_t>{-3, -2}),      // Axes
+                       ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+const auto testCaseRepeatingAxes5D =
+    ::testing::Combine(::testing::Values(std::vector<size_t>{2, 16, 32, 7, 32}),                    // Input shape
+                       ::testing::ValuesIn(inputPrecision),                                         // Precision
+                       ::testing::Values(std::vector<int64_t>{16, 15, 10, 2, 1, 7, 2, 8, 1, 1}),    // Shift
+                       ::testing::Values(std::vector<int64_t>{-1, -2, -3, 1, 0, 3, 3, 2, -2, -3}),  // Axes
+                       ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+const auto testCaseNegativeShifts6D =
+    ::testing::Combine(::testing::Values(std::vector<size_t>{4, 16, 3, 6, 5, 2}),         // Input shape
+                       ::testing::ValuesIn(inputPrecision),                               // Precision
+                       ::testing::Values(std::vector<int64_t>{-2, -15, -2, -1, -4, -1}),  // Shift
+                       ::testing::Values(std::vector<int64_t>{0, 1, 2, 3, 4, 5}),         // Axes
+                       ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+const auto testCaseUnordNegAxesAndShifts10D =
+    ::testing::Combine(::testing::Values(std::vector<size_t>{2, 2, 4, 2, 3, 6, 3, 2, 3, 2}),  // Input shape
+                       ::testing::ValuesIn(inputPrecision),                                   // Precision
+                       ::testing::Values(std::vector<int64_t>{-2, -1, 1, 1, 1, -2}),          // Shift
+                       ::testing::Values(std::vector<int64_t>{-6, -4, -3, 1, -10, -2}),       // Axes
+                       ::testing::Values(CommonTestUtils::DEVICE_GPU));
+
+INSTANTIATE_TEST_SUITE_P(smoke_INTEL_GPU_TestsRoll_2d_zero_shifts,
```

Comment:
I think INTEL_GPU is excessive in the testcase name. Anyway this code will be compiled into GPU tests binary.

Reply:
Nice point, I just copied from CPU tests, don't check, done